frontend/src/Movie/Details/MovieDetails.js

```diff
@@ -282,7 +282,8 @@ class MovieDetails extends Component {
     } = this.state;
 
     const marqueeWidth = isSmallScreen ? titleWidth : (titleWidth - 150);
-    const friendlyRuntime = moment.utc((runtime * 60) * 1000).format('h[h] m[m]');
+    const movieMicroTime = (runtime * 60) * 1000;
+    const friendlyRuntime = moment.utc(movieMicroTime).format((movieMicroTime < 3600000) ? 'm[m]' : 'h[h] m[m]');
```

Comment:
Can we just purge the use of moment here and create a native function in Utilities... Seems to be about 50 times faster doing something like below vs using moment.js which will be nicer when it comes to index performance.
```
const hours = Math.floor(runtime / 60);
runtime -= (hours * 60);
const minutes = runtime;
const formatedRuntime = hours > 0 ? `${hours}h ${minutes}m` : `${minutes}m`;
```

Reply:
I didn't use moment originally, I had my own way to do it & you said to use moment.. Ffs lol